app/lib/notifications.js

```diff
@@ -5,6 +5,9 @@ const slackLib = require('./slack');
 const ACTIVITY_ALL = 'all';
 
 module.exports = (Sequelize, activity) => {
+  if(typeof activity.GroupId === 'undefined' || typeof activity.type === 'undefined') {
```

Comment:
Are you sure that `GroupId` is `undefined` and not `null`?
Have you tested this?
We pushed this to prod and it doesn't seem to work.